gatsby/src/docs/sdks/javascript/config/basics.mdx

```diff
@@ -51,6 +51,9 @@ These options can be read from an environment variable - `SENTRY_DSN`, `SENTRY_E
 
 : A list of strings or regex patterns that match error URLs that should exclusively be sent to Sentry. This is a "contains" match to the entire file URL. As a result, if you add `foo.com` to it, it will also match on `[https://bar.com/myfile/foo.com](https://bar.com/myfile/foo.com)` By default, all errors will be sent.
 
+`normalizeDepth`
+
+: Sentry SDKs normalize any context to a depth of 3.
```

Comment:
```suggestion
: Sentry SDKs normalize any contextual data to a given depth. Any keys containing data with a structure deeper than this will be trimmed and marked using its type instead (`[Object]` or `[Array]`), without walking the tree any further. By default, walking is performed `3` levels deep.
```